### DISCUSSION TITLE
How to configure the network in order to make Frontend and Backend apps work?

### DISCUSSION THREAD

**Discussion by kamzata** [dokku/dokku#4932] [Q&A] [answered]
How can I set the network in order to make the Frontend app "talks" to backend (which should be answer at http://localhost:3900)?

I found [this post which use Docker](https://aviaryan.com/blog/dokku-access-app-internally) but I think it's a little bit dated and now I can use just Dokku, can't I?

**Comment by Cellane** [comment, marked as answer]
Sure you can! Have a look at [this page in the documentation](https://dokku.com/docs/networking/network/).

(Bear in mind that none of this applies if your front-end is a single-page app where the communication with the back-end is initiated via JavaScript from the user’s web browser.)
